runtime/plugin/health.vim

```diff
@@ -1 +1,7 @@
-command! -nargs=* CheckHealth call health#check([<f-args>])
+function! s:complete(lead, _line, _pos) abort
+  return filter(map(glob($VIMRUNTIME.'/autoload/health/*', 1, 1),
```

Comment:
And note: this is *not* about supporting Windows. This is about supporting special characters in $VIMRUNTIME directory name and linux allows having much more of them.